test/dummy.test.ts

```diff
@@ -0,0 +1,10 @@
+import {add} from "../src/dummy/dummy";
```

Comment:
I guess you could use "dummy/dummy" here, since "./src" is the base path.

Reply:
Changed.